test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,45 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() {
+
+        // some null
+        assertHasSomeNull((Object) null);
+        assertHasSomeNull(null, null);
+        assertHasSomeNull(null, "abc", "abc");
+        assertHasSomeNull(null, "abc", "", "abc", "ABC");
+        assertHasSomeNull(null, "", "abc", "a", "abc");
+        assertHasSomeNull(null, 1, new Integer(1));
+        assertHasSomeNull(null, null, 1, new Integer(1));
+        assertHasSomeNull(null, null, null);
+        assertHasSomeNull(null, null, "a", "b", null);
+
+        // no null
+        assertNoNull(1);
+        assertNoNull("");
+        assertNoNull("abc");
+        assertNoNull("abc", "ab", "a");
+        assertNoNull(1, 2);
+        assertNoNull("abc", "abc");
+        assertNoNull("abc", "", "abc", "ABC");
+        assertNoNull("", "abc", "a", "abc");
+        assertNoNull(1, new Integer(1));
+    }
+
     private void assertAreUnique(Object... objects) {
         assertTrue(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
 
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertHasSomeNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
+    }
+
+    private void assertNoNull(Object... objects) {
```

Comment:
Try keeping a consistent naming convention: `assertHasNoNull` or changing the above method to `assertSomeNull`.